sdks/python/apache_beam/io/external/kafka.py

```diff
@@ -37,18 +37,25 @@
 
 from __future__ import absolute_import
 
-from apache_beam import ExternalTransform
-from apache_beam import pvalue
-from apache_beam.coders import BytesCoder
-from apache_beam.coders import IterableCoder
-from apache_beam.coders import TupleCoder
-from apache_beam.coders.coders import LengthPrefixCoder
-from apache_beam.portability.api.external_transforms_pb2 import ConfigValue
-from apache_beam.portability.api.external_transforms_pb2 import ExternalConfigurationPayload
-from apache_beam.transforms import ptransform
-
-
-class ReadFromKafka(ptransform.PTransform):
+import typing
+
+from past.builtins import unicode
+
+from apache_beam.transforms.external import ExternalTransform, NamedTupleBasedPayloadBuilder
+
+
+ReadFromKafkaSchema = typing.NamedTuple(
```

Comment:
btw, I used the name "schema" since ultimately we will switch to using the upcoming schema support when serializing this payload.  Not sure if that changes your mind on the naming at all.